### DISCUSSION TITLE
How to add content created in postUpgradeTasks to the prBodyNotes?

### DISCUSSION THREAD

**Discussion by bitschubse** [renovatebot/renovate#32007] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure DevOps

### Please tell us more about your question or problem

Hi all,
we generate additional information about the update in the `postUpgradeTasks`. We would like to display this data in the pull request, e.g. in the `prBodyNotes`.
Is there a way to do this? Is it possible to use your own dynamic content in the templating that goes beyond the fields provided?

**Comment by rarkins** [comment, marked as answer]
It's not possible to extend the data in the way you need

**Reply by bitschubse** [reply]
@rarkins: Thanks for the quick answer. Would it be possible in principle to implement such a feature or is it architecturally impossible?

**Reply by rarkins** [reply]
I think it's architecturally possible if you want to attempt it
